runtime/caml/config.h

```diff
@@ -153,6 +153,8 @@ typedef uint64_t uintnat;
 #error "No integer type available to represent pointers"
 #endif
 
+#define UINTNAT_MAX -1
```

Comment:
Grmph...  what about `(uintnat) -1` ?  At least it makes the type and the value explicit.